src/ucs/vfs/sock/vfs_sock.h

```diff
@@ -7,8 +7,83 @@
 #ifndef UCS_VFS_SOCK_H_
 #define UCS_VFS_SOCK_H_
 
+#include <sys/types.h>
+#include <sys/un.h>
+#include <stdint.h>
+
 /* This header file defines socket operations for communicating between UCS
  * library and VFS daemon */
 
+/**
+ * VFS socket message type
+ */
+typedef enum {
+    UCS_VFS_SOCK_ACTION_STOP, /* daemon is asked to stop */
```

Comment:
align comments?

Reply:
currently clang-format does not do struct members comment alignment very well, but i guess better align and fix the formatter

Reply:
> currently clang-format does not do struct members comment alignment very well, but i guess better align and fix the formatter

ic, thanks for the explanation